src/sdf_tools/bindings.cpp

```diff
@@ -0,0 +1,53 @@
+#include <cstdint>
+
+#include <pybind11/eigen.h>
+#include <pybind11/functional.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "sdf_tools/collision_map.hpp"
+
+namespace py = pybind11;
+using namespace Eigen;
+using namespace sdf_tools;
+
+PYBIND11_MODULE(pysdf_tools, m)
+{
+  Py_Initialize();
+
+  // Bindings
+  py::class_<COLLISION_CELL>(m, "COLLISION_CELL")
+      .def(py::init<float>())
+      .def(py::init<float, uint32_t>())
+      .def_readwrite("occupancy", &COLLISION_CELL::occupancy)
+      .def_readwrite("component", &COLLISION_CELL::component);
+
+  py::class_<Isometry3d>(m, "Isometry3d").def(py::init<Matrix4d>());
```

Comment:
Move this to an `arc_utilities` python library?

Reply:
we can do that later, noted for the future.